sentry-spring-jakarta/src/main/java/io/sentry/spring/jakarta/SentrySpringFilter.java

```diff
@@ -31,15 +31,15 @@
 
 @Open
 public class SentrySpringFilter extends OncePerRequestFilter {
-  private final @NotNull IScopes scopes;
+  private final @NotNull IScopes scopesBeforeForking;
```

Comment:
Trying to make it more clear, this should only be used for forking and checks but not for writing things like breadcrumbs, starting transaction etc.